sgx_tcrypto_helper/src/rsa2048.rs

```diff
@@ -0,0 +1,447 @@
+use crypto::rsgx_create_rsa_key_pair;
```

Comment:
Can you add the Apache license header also?

Reply:
Sure. I have added the apache license header. And also add the LICENSE file that other crates have in the root folder.